src/scenarios/personalization/ProgressSet/ProgressSet.tsx

```diff
@@ -0,0 +1,181 @@
+import React, { useState, useRef, useEffect, Component } from 'react';
+import {
+  Button,
+  Container,
+  Divider,
+  Form,
+  Grid,
+  Header,
+  Label,
+  Segment,
+} from 'semantic-ui-react';
+import {
+  useScenarioHost,
+  VariableSearch,
+} from '@axinom/mosaic-fe-samples-host';
+import { getApolloClient } from '../../../apollo-client';
+import { SetProgressMutation } from './graphql-documents';
+
+export const ProgressSet: React.FC = () => {
+  const { activeProfile, logger } = useScenarioHost();
+  const [FrequencyOfProgressSaving, setFrequencyOfProgressSaving] =
+    useState<number>();
+  const [userAccessToken, setUserAccessToken] = useState<string>();
+  const [key, setKey] = useState<string>();
+  const [buttonState, setButtonState] = useState<boolean>(true);
```

Comment:
Shall we use a more meaningful name for this state? i'd recommend that we call it `isRunning` (opposite of how you are currently using the boolean - so the default value should now be false) and update the logic accordingly.